src/tox/venv.py

```diff
@@ -266,11 +266,11 @@ def _getliveconfig(self):
         alwayscopy = self.envconfig.alwayscopy
         deps = []
         for dep in self.get_resolved_dependencies():
-            dep_name_md5 = getdigest(dep.name)
-            deps.append((dep_name_md5, dep.name))
-        base_resolved_python_md5 = getdigest(base_resolved_python_path)
+            dep_name_sha256 = getdigest(dep.name)
```

Comment:
If we call such variables just `dep_name_hash` etc., the name would not need to change again when `getdigest` is updated (for example once sha256 is considered "old and dangerous" in 10 years from now) :thinking:

Reply:
I don't expect this code to be around in 10 years 👍